### PR TITLE
dump/load

### PR DESCRIPTION
**Pull request by barakalon**
Dump/load an AST to/from JSON

Context: https://tobiko-data.slack.com/archives/C0448SFS3PF/p1673005789088219

### CODE REVIEW

File: sqlglot/serde.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+from sqlglot import expressions as exp
+
+
+def dump(node):
```
Comment (tobymao) [inline sqlglot/serde.py:4]: add types

**Review by GeorgeSittas — APPROVED** [review]
Looks good, what about a test for comments too, for the sake of completeness?

**Comment by barakalon** [comment]
The identity tests include comments

**Comment by GeorgeSittas** [comment]
Right, my bad. I thought we only had them under tests for pretty. All good.

File: sqlglot/serde.py (hunk 1 above)
Comment (barakalon) [reply sqlglot/serde.py]: Kind of a weird thing to annotate... but I gave it a shot

**Review by tobymao — APPROVED** [review]

**Pull request merged**